PowerPointLabs/PowerPointLabs/Models/PowerPointCurrentPresentationInfo.cs

```diff
@@ -57,5 +63,35 @@ public static IEnumerable<PowerPointSlide> SelectedSlides
                 return slides;
             }
         }
+
+        private static bool ActivateWindow() 
+        {
+            try 
+            {
+                Globals.ThisAddIn.Application.Activate();
+                if (Globals.ThisAddIn.Application.Active == Microsoft.Office.Core.MsoTriState.msoTrue && Globals.ThisAddIn.Application.Windows.Count >= 1)

```

Comment:
In Ln 71 the application has been activated, hence it would always be true in Ln 72 for the condition of

`Globals.ThisAddIn.Application.Active == Microsoft.Office.Core.MsoTriState.msoTrue`

Reply:
I’m afraid for some reason the API does not work and the application does not become active. But on second thought, if the API does not work it’ll probably throw a COMexception and it will be caught. I’ll check that.